Sources/Gridicons/GridiconsGenerated.swift

```diff
@@ -412,6 +413,19 @@ extension GridiconType {
   }
 
   var icon: UIImage {
+    let image = UIImage(named: name, in: bundle(), compatibleWith: nil)
```

Comment:
Are these changes added manually or have you not committed `.stencil` file?

I understand:
1. This code has to be added to `Gridicons.stencil`
2. Run `rake gen` to generate a new `GridiconsGenerated.swift` file
3. Confirm that `GridiconsGenerated.swift` resembles what you have here

Reply:
Thanks for noticing that this is generated from a template! Updated in https://github.com/Automattic/Gridicons-iOS/pull/82/commits/57dde40fcca66115a3fb8d1272686f065f5a63f3.

I was wondering why it was called *Generated but didn't see the "do not edit" note, and didn't see any notes about development in the README, I thought maybe just the icons were generated. Added a development section about the generated file in https://github.com/Automattic/Gridicons-iOS/pull/82/commits/5ed68663d62cf09694f791cf5cc2c317537701ff.